components/Nav/NavMobile/NavMobile.js

```diff
@@ -34,7 +34,7 @@ export default class NavMobile extends Component {
     return (
       <header className={styles.NavMobile}>
         <Link href="/" prefetch={false}>
-          <button
+          <button role='button' aria-label='Mobile Navigation Button'
```

Comment:
I'm glad you touched this so I can see a new problem!

This should be `a` tag, not a button. After you change it, you can ditch the role, name, and type attributes from this element.

In terms of [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute), you want to describe the action the occurs when clicking on the link. Since this link navigates the user to the home page, `Go Home` seems like a good choice.